### PR TITLE
Ignore audio errors instead of crashing

### PR DESCRIPTION
**Pull request by Emiluren**

### CODE REVIEW

**Comment by Emiluren** [comment]
This is ok to be rebased

**Review by malcx95 — APPROVED** [review]

**Comment by TheZoq2** [comment]
I think we should print the errors if they happen, but not crash the program

**Comment by Emiluren** [comment]
> I think we should print the errors if they happen, but not crash the program

Yes I was considering this. I can change it.

**Comment by Emiluren** [comment]
I was afraid that there would be a lot of spam

**Comment by Emiluren** [comment]
@TheZoq2 pls check again

File: src/client.rs [hunk 1]
```diff
@@ -81,32 +81,30 @@ impl MainState {
                     self.game_state = state
                 },
                 ServerMessage::PlaySound(sound, pos) => {
+                    fn play_sound(soundeffect: &sdl2::mixer::Chunk) {
+                        if let Err(e) = sdl2::mixer::Channel::all().play(
+                            soundeffect, 0
+                        ) {
```
Comment (TheZoq2) [inline src/client.rs:87]: Weird indentation, but I guess that's fine

**Review by TheZoq2 — APPROVED** [review]

File: src/client.rs (hunk 1 above)
Comment (Emiluren) [reply src/client.rs:87]: No you're wrong :P

Comment (TheZoq2) [reply src/client.rs:87]: :(

**Pull request merged**